src/main/java/ch/wisv/areafiftylan/products/model/TicketOption.java

```diff
@@ -19,39 +19,28 @@
 
 import ch.wisv.areafiftylan.utils.view.View;
 import com.fasterxml.jackson.annotation.JsonView;
-import lombok.EqualsAndHashCode;
-import lombok.Getter;
-import lombok.NoArgsConstructor;
-import lombok.Setter;
+import lombok.*;
 
 import javax.persistence.Entity;
 import javax.persistence.GeneratedValue;
 import javax.persistence.Id;
 
 @Entity
+@Data
+@RequiredArgsConstructor
 @NoArgsConstructor
-@EqualsAndHashCode
 public class TicketOption {
 
     @Id
     @GeneratedValue
-    @Getter
     private Long id;
 
-    @Getter
-    @Setter
-    @JsonView(View.OrderOverview.class)
-    private float price;
-
-    @Getter
-    @Setter
+    @NonNull
```

Comment:
https://projectlombok.org/features/Constructor.html
Lombok's NonNull injects a null check, and marks that field to get a parameter in the @RequiredArgsConstructor.
The latter is why it's used here.